src/server.h

```diff
@@ -115,6 +115,7 @@ struct hdr_histogram;
 #define CONFIG_MAX_HZ            500
 #define MAX_CLIENTS_PER_CLOCK_TICK 200          /* HZ is adapted based on that. */
 #define CRON_DBS_PER_CALL 16
+#define CRON_DICTS_PER_DB 16
```

Comment:
I did keep only one loop as the slow scan.
`CRON_DICTS_PER_DB` is meant to be used in cluster mode, where one DB has 16384 dicts. If we only resize 1 dict per cron, it would be too slow.
And in standalone mode, `tryResizeHashTables` would resize one dict/DB per cron.

Reply:
isn't that what we had for "fast" scan until a few days ago? and the "slow" scan this PR introduced handled just one dict per cron?
i'm not sure it'll be too slow, considering it's only meant to handle changes that were made to a dict during a fork (where resizing is suspended), and then after the fork was done, all changes to that dict stopped (otherwise the traffic would trigger the resize, and cron isn't needed).
i thought that's the conclusion we made in the core-team meeting. @soloestoy correct me if i'm wrong

Reply:
Let's set aside the notions of "fast" and "slow" for now because there is no distinction between them anymore. In fact, we are discussing how to handle the automatic expand and shrink that is blocked by a fork.

Therefore, in this fallback solution, it is sufficient to check one database at a time. @CharlesChen888  please make the necessary modifications accordingly.

Taking into account the differences between standalone and cluster, where standalone has only one dict per database while cluster has 16,384 dicts per database, I think it's reasonable to use `CRON_DICTS_PER_DB`.

Reply:
Let's set aside the notions of "fast" and "slow" for now because there is no distinction between them anymore. In reality, we are discussing how to handle the automatic expansion and shrinkage that is blocked by a fork.

Therefore, in this fallback solution, it is sufficient to check one database at a time. @CharlesChen888 please make the change accordingly.

Taking into account the differences between standalone and cluster, where standalone has only one dictionary per database while cluster has 16,384 dictionaries per database, I think it's reasonable to use `CRON_DICTS_PER_DB` for cluster.

Reply:
i was under the understanding that we agreed that the rate can be reduced, but i'm fine with keeping it.